### PR TITLE
fix: fix spawning Appium on Windows

### PR DESCRIPTION
**Pull request by wswebcreation**
## Proposed changes
This fixes spawning Appium on Windows, see https://github.com/webdriverio/webdriverio/issues/4914

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments
Need to add tests for it, first wanted to check if you think this is the best way to do it

### CODE REVIEW

**Comment by mgrybyk** [comment]
I think we did something similar for selenium standalone service. 
Mind adding unit tests?

**Comment by wswebcreation** [comment]
Sure, I want to add this one

```js
        test('should set correct config properties for Windows', async () => {
            const config = {
                appium: {
                    logPath: './',
                    command: 'path/to/my_custom_appium',
                    args: { foo: 'bar' }
                }
            }
            await launcher.onPrepare(config)

            expect(launcher.command).toBe('cmd')
            expect(launcher.appiumArgs).toEqual(['/c', config.appium.command, '--foo', 'bar'])

        })
```

But I don't know how to mock the `process.env` here so I can hit line 54

File: packages/wdio-appium-service/src/launcher.js [hunk 1]
```diff
@@ -48,8 +49,12 @@ export default class AppiumLauncher {
     }
 
     _startAppium(command, args, waitStartTime, callback) {
-        log.debug(`Will spawn Appium process: ${command} ${args.join(' ')}`)
-        let process = spawn(command, args, { stdio: ['ignore', 'pipe', 'pipe'] })
+        const cmd = isWindows ? 'cmd' : command
```
Comment (baruchvlz) [inline packages/wdio-appium-service/src/launcher.js]: Is this the best place to do this? It seems like you'll be overwriting any command passed through the [onPrepare method](https://github.com/webdriverio/webdriverio/blob/master/packages/wdio-appium-service/src/launcher.js#L24-L30).

Comment (wswebcreation) [reply packages/wdio-appium-service/src/launcher.js]: I think so, Windows commands need to be started different in comparison to Mac/Linux

Comment (christian-bromann) [reply packages/wdio-appium-service/src/launcher.js]: @wswebcreation I think what @baruchvlz was saying is whether we could move this to `onPrepare`

File: packages/wdio-appium-service/src/launcher.js [hunk 2]
```diff
@@ -48,8 +49,12 @@ export default class AppiumLauncher {
...
+        if(isWindows){
+            args.unshift('/c', command)
```
Comment (erwinheitzman) [inline packages/wdio-appium-service/src/launcher.js]: I'd prefer to use `path.parse(process.cwd()).root` to identify the operating system's root but it's not mandatory I suppose

File: packages/wdio-appium-service/src/launcher.js (hunk 1 above)
Comment (wswebcreation) [reply packages/wdio-appium-service/src/launcher.js]: Moved it, also gave me the option to mock the `process.platform`

**Pull request merged**